definitions/prepare-validation/prepare-validation.yaml

```diff
@@ -0,0 +1,22 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: prepare-validation
+spec:
+  params:
+    - name: applicationSnapshot
+      type: string
+      description: The ApplicationSnapshot in JSON format to apply the mapping to
+  results:
+    - name: applicationSnapshot
```

Comment:
@davidmogar I used this name to keep the m5 release pipeline pretty looking. Let me know if you would rather me rename it